bolts/data/datasets/utils.py

```diff
@@ -108,6 +112,23 @@ def img_to_tensor(img: Image.Image | np.ndarray) -> Tensor:
     )
 
 
+AudioLoadingBackend = Literal["sox_io", "soundfile"]
+
+AudioTform = Union[TF_audio.Spectrogram, TF_audio.MelSpectrogram]
```

Comment:
There's no base class for torch audio transforms but they do all have the common property of taking in a tensor as input and returning a tensor. You can annotate this using typing.Callable.

Reply:
Ah yes, that is much clearer I didn't know about the Callable, fixed now.